src/routes/index.jsx

```diff
@@ -1,22 +1,31 @@
 import { createBrowserRouter } from 'react-router-dom';
 import Dashboard from '../pages/user/Dashboard';
 import ProductDetails from '../pages/user/ProductDetails';
+import Signin from '../pages/auth/Signin';
```

Comment:
create index.js and add SignIn page there. Expected input here is:
import { Signin } from '../pages';